@SebastianM-C  Thanks for your PR. Are you sure the it builds for you though? For me it's been failing at `package` as follows:

```
==> Entering fakeroot environment...
==> Starting package()...
 INSTALL /usr/bin/singularity
install: cannot remove '/usr/bin/singularity': Permission denied
make: *** [Makefile:307: /usr/bin/singularity] Error 1
==> ERROR: A failure occurred in package().
    Aborting...
```

3.0.0 doesn't have this problem. Looks like the upstream has again changed something in their build process. Maybe `PREFIX` in `make PREFIX="${pkgdir}/usr" install` is not being honored anymore? Needs investigation. Please let me know if you are going to look into this.

--- a/PKGBUILD
+++ b/PKGBUILD
@@ -22,7 +22,7 @@ prepare() {
 build() {
   export GOPATH="${srcdir}/singularity"
   cd "${GOPATH}/src/github.com/sylabs/singularity"
-  ./mconfig -p /usr
+  ./mconfig --prefix="${pkgdir}/usr"
   cd builddir
   make
 }

